samples/RewriteSample/Startup.cs

```diff
@@ -26,17 +26,19 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment hostingEnv)
             // (ex StringReplace) that are easy to implement in code, they can do so by calling 
             // AddFunctionalRule(Func);
             // TODO make this startup do something useful.
+
             app.UseRewriter(new RewriteOptions()
+                .Rewrite(@"foo/(\d+)", "foo?id={R:1}") 
                 .ImportFromUrlRewrite(hostingEnv, "UrlRewrite.xml")
-                .ImportFromModRewrite(hostingEnv, "Rewrite.txt")
-                .RedirectToHttps(StatusCodes.Status307TemporaryRedirect)
-                .RewriteRule("/foo/(.*)/bar", "{R:1}/bar")
-                .AddRule(ctx =>
-                {
-                    ctx.HttpContext.Request.Path = "/index";
-                    return RuleResult.Continue;
-                }));
-
+                .ImportFromModRewrite(hostingEnv, "Rewrite.txt"));
+            // Regex syntax instead
+            //.Redirect(@"bar", "foo/10")                                     
```

Comment:
Is this PR will WIP? Otherwise let's not leave commented code behind. If they're intentionally commented out as sample, add proper comments to explain them to the user.